stko/optimizers/collapser.py

```diff
@@ -149,70 +152,70 @@ def _get_new_position_matrix(self, mol, step, vectors, scales):
 
         return new_position_matrix
 
-    def _get_BB_vectors(self, mol, BB_atom_ids):
+    def _get_bb_vectors(self, mol, bb_atom_ids):
         """
-        Get the BB to COM vectors.
+        Get the bb to COM vectors.
```

Comment:
How does the user know what 'bb' is? Maybe change to building block.

Reply:
Fixed.